httpsig_test.go

```diff
@@ -616,6 +807,17 @@ func loadPrivateKey(keyData []byte) (*rsa.PrivateKey, error) {
 	return x509.ParsePKCS1PrivateKey(pem.Bytes)
 }
 
+// taken from https://blainsmith.com/articles/signing-jwts-with-gos-crypto-ed25519/
+func loadEd25519PrivateKey(keyData []byte) (ed25519.PrivateKey, error) {
+	var block *pem.Block
+	block, _ = pem.Decode(keyData)
+
+	var asn1PrivKey ed25519PrivKey
+	asn1.Unmarshal(block.Bytes, &asn1PrivKey)
+
+	return ed25519.NewKeyFromSeed(asn1PrivKey.PrivateKey[2:]), nil
```

Comment:
`asn1PrivKey.PrivateKey[2:]` is some ASN1 octet string dark magic, do you mind adding a comment noting that `[2:]` is skipping the byte for TAG and the byte for LEN?